makefile

```diff
@@ -55,6 +55,10 @@ ARCH_is_$(ARCH)                    := yes
 DEFAULT_BUILD_PARAMETERS_LIB       := $(if $(OS_is_win),no,yes)
 BUILD_PARAMETERS_LIB               ?= $(DEFAULT_BUILD_PARAMETERS_LIB)
 
+ifeq ($(RNG_BACKEND), openrng)
+	RNG_OPENRNG := yes
+endif
```

Comment:
RNG_BACKEND variable should be one of `mkl`, `ref` and `openrng` with default value dependent on platform type.